pkg/client/unversioned/discovery_client.go

```diff
@@ -116,21 +92,17 @@ func (d *DiscoveryClient) ServerGroups() (apiGroupList *unversioned.APIGroupList
 
 // ServerResourcesForGroupVersion returns the supported resources for a group and version.
 func (d *DiscoveryClient) ServerResourcesForGroupVersion(groupVersion string) (resources *unversioned.APIResourceList, err error) {
-	url := d.baseURL
+	url := url.URL{}
 	if groupVersion == "v1" {
```

Comment:
Does `/api/v1` return a list of server resources in 1.0? Is this expected to work against 1.0 servers?